TWLight/resources/models.py

```diff
@@ -430,7 +428,11 @@ def clean(self):
             )
             if validation_error_msg:
                 raise ValidationError({"target_url": validation_error_msg})
-
+        if self.PROXY or self.BUNDLE and not self.target_url:
+            raise ValidationError(
+            'Proxy and Bundle partners require a target URL'
+            )
+            
```

Comment:
Suggested at [Phabricator task T236390](https://phabricator.wikimedia.org/T236390)
Although I do not know the definition of `ValidationError` here.
```python
if self.PROXY or self.BUNDLE and not self.target_url:
    raise ValidationError('Proxy and Bundle partners require a target URL')